customers extended of billing code and accounting code

MartinPrazak opened the pull request:
customers extended of billing code and accounting code

merge after https://github.com/MewsSystems/mews/pull/14157

vitasamek commented:
@MartinPrazak Can you please through all your PR and prepare release notes?
Look at the structure of the Changelog. It will save a lot of time for me.

MartinPrazak commented:
```
## 7th October 2019 12:42 UTC

* Extended [Customer](operations/customers.md#customer) response with BillingCode and AccountingCode
* Extended [Configuration](operations/configuration.md#configuration) response with [PaymentCardStorage](operations/configuration.md#payment-card-storage).
* Added [Add tokenized credit card](operations/finance.md#add-tokenized-credit-card) operation.
* Added [Add space block](operations/enterprises.md#add-space-block) operation.
```

siroky commented on operations/customers.md:
```diff
@@ -90,6 +90,8 @@ Returns all customers from the specified interval according to the time filter \
 | `Phone` | string | optional | Phone number of the customer \(possibly mobile\). |
 | `TaxIdentificationNumber` | string | optional | Tax identification number of the customer. |
 | `LoyaltyCode` | string | optional | Loyalty code of the customer. |
+| `AccountingCode` | string | optional | Accounting code of the customer. |
```
@vitasamek you might have forgotten to update some JSONs with customers.
